internal/ui/settings_ui.go

```diff
@@ -378,6 +378,25 @@ func getPicturesDir() (fyne.ListableURI, error) {
 		return nil, err
 	}
 
-	uri := storage.NewFileURI(filepath.Join(home, "Pictures"))
-	return storage.ListerForURI(uri)
+	if runtime.GOOS == "darwin" {
```

Comment:
Here we set up a darwin branch for the generic solution, then assume xdg for any other system... So if someone were running this on Windows, or Linux without the xdg binary (totally possible) it will fail with an error.

Why not invert the logic - if the exec.LookPath finds xdg-user-dir then use it, otherwise take the generic approach?

Reply:
Sorry. I did not think that fynedesk was runnable on Windows due to relying on Xorg among other stuff.
That sounds like a great approach :+1: